Lib/test/test_threading.py

```diff
@@ -1010,7 +1010,7 @@ def callback():
             threading.settrace(noop_trace)
 
             # Create a generator in a C thread which exits after the call
-            import _testcapi
+            _testcapi = import_module("_testcapi")
```

Comment:
Move it at the top of the function.

Reply:
Resolved in 88c6739d3a19992cf8ca8fc36190b66635a5f99a